.travis.yml

```diff
@@ -32,18 +32,9 @@ before_script:
 
 script: make check && make test
 
-before_deploy:
-- make compile-catalog
-
 deploy:
-- provider: pypi
-  server: https://upload.pypi.org/legacy/
-  user: sbrunner
-  password:
-    secure: "PoIuVu4sdApkWm6CXnBMMzXCj1vNnEdLXKqcBkXlKjBxh6JlivGRmqoFizJAe3KNjZMwMBMcJkMR+f53g0z3P06sIx9WQCaAjYV0j5w5Si71tjjOWohY2DjIaUcNNOsodZc9KDdtSpN0Xm4+vbn7Lg9FofCovs2KAIsBONVD/I4="
-  distributions: sdist bdist_wheel
-  skip_upload_docs: true
-  skip_cleanup: true
+- provider: script
+  script: travis/deploy-daily-pypi
```

Comment:
Seems that this do not match with new script name after renaming.

Reply:
Oups, thanks, fixed :-)